instrumentation/opentelemetry/init.go

```diff
@@ -92,6 +96,14 @@ func makeExporterFactory(cfg *config.AgentConfig) func() (sdktrace.SpanExporter,
 			opts = append(opts, otlpgrpc.WithInsecure())
 		}
 
+		if len(cfg.GetReporting().GetCertFile().GetValue()) > 0 {
```

Comment:
`cfg.GetReporting().GetCertFile().GetValue()` is being called three times in this code, I believe we should move it into a variable.

Reply:
fixed